eip-0027.md

```diff
@@ -113,11 +107,15 @@ Contracts
     val correctCoinsIssued = EQ(reemissionRewardPerBlock, Minus(ExtractAmount(Self), ExtractAmount(reemissionOut)))
     
     // when reemission contract box got merged with other boxes
-    val sponsored = AND(
+    // TODO: describe why this condition can replace 5 others (heightCorrect, etc)
```

Comment:
it is not replacing them, rather, an alternative branch. So re-emission box is whether getting additional funding via `sponsored` path, or spending funds according to rules (heightCorrect etc)

Reply:
I don't understand what `sponsored` means, what is the intuition about this. Better to put it into the contract. 
Without understanding the intended logic, it is hard to review and estimate correctness of the contract.

Reply:
Contract description in this EIP have this: 
```
Also, this contract allows for merging with other boxes; the merging transaction must have only two outputs; the first output is for the re-emission contract, the second one is to pay a mining fee (its value can be 0.01 ERG at most)
```

Reply:
Ok, I've read that. Why not also hint about what are the inputs of the merging transaction?

Reply:
Ok, the comment above inputs would be helpful in the code to decipher the contract.